src/Stryker.Core/Stryker.Core/Mutators/StatementMutator.cs

```diff
@@ -0,0 +1,110 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+
+namespace Stryker.Core.Mutators
+{
+    public class StatementMutator: MutatorBase<StatementSyntax>, IMutator
+    {
+        public override MutationLevel MutationLevel => MutationLevel.Standard;
+
+        private static readonly HashSet<SyntaxKind> AllowedSyntaxes = new HashSet<SyntaxKind>()
+        {
+            // SyntaxKind.EmptyStatement, // useless mutation
+
+            // SyntaxKind.Block, // unitary mutation should prevail over blocks
+            // SyntaxKind.CheckedStatement, // not unitary
```

Comment:
Could you clarify what the comment not unitary means on these types of statements

Reply:
As "not unitary" I meant every statement that contains more statements inside (usually, containing blocks)